tasks/setup-Debian.yml

```diff
@@ -1,4 +1,45 @@
 ---
+- name: Ensure the WebUpd8 repo key is present.
```

Comment:
This makes sense. 
I'll try https://github.com/geerlingguy/ansible-role-repo-dotdeb to install WebUpd8 and my own Playbook to install Oracle Java.
Regards.